chainer/links/caffe/caffe_function.py

```diff
@@ -318,6 +319,54 @@ def _setup_relu(self, layer):
         self.forwards[layer.name] = fw
         self._add_layer(layer)
 
+    @_layer('Eltwise', 'ELTWISE')
+    def _setup_eltwise(self, layer):
+        operation = layer.eltwise_param.operation
+        coeffs = layer.eltwise_param.coeff
+        if coeffs == []:
+            coeffs = None
+
+        self.forwards[layer.name] = _EltwiseFunction(operation, coeffs)
+        self._add_layer(layer)
+
+    @_layer('Scale', 'NONE')    # TODO: old name
+    def _setup_scale(self, layer):
```

Comment:
Please write the note that we do not support `filler`, `bias_filler` of `scale_param`. (I think we do not need to support them until we truly need it).